avocado_vt/plugins/vt.py

```diff
@@ -250,6 +250,20 @@ def configure(self, parser):
             help=help_msg,
         )
 
+        help_msg = (
+            "Omits the `OSError [Errno 9] Bad file descriptor` caused by "
+            "avocado.utils.process utility during clean up. This can be used "
+            "when this error would cause false positive failures of a test."
+        )
+        add_option(
```

Comment:
if introduce new option in avocado run (here --vt-omit-data-loss), it will bring additional efforts to update our CI to enable that. I wonder whether we can introduce one gating_variable e.g omit_data_loss in avocado_vt/conf.d/vt.conf, if omit_data_loss is set true, it enable it implicitly .

Reply:
With this PR, you can enable it with config option as well by setting `vt.omit_data_loss` to true. see https://github.com/avocado-framework/avocado-vt/pull/4001/files#diff-64c70a40c0700575186086930678d178f26a5b3fbe71dfe9d1ebd20d4d5e7752R135 

But I am not sure if we should enable this implicitly, because it can hide real failures by skipping error messages. AFAIK this issue is related only to your CI and I think we shouldn't affect others by this.